Apsitvarkom.Models/PollutedLocation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using static Apsitvarkom.Models.Enumerations;
+
+namespace Apsitvarkom.Models;
+
+/// <summary>
+/// Class for storing captured polluted location records.
+/// </summary>
+public class PollutedLocation
+{
+    private int _progress;
+    private int _radius;
+
+    /// <summary>Unique identifier of the given record.</summary>
+    public string Id { get; init; } = null!;
```

Comment:
is this not of type `Guid`?

Reply:
Initially I used Gui, but then I figured we could save it as strings instead of Guid to simplify its storing while still maintaining its offered uniqueness.

Reply:
We will save it as strings when it's actually saved somewhere but if we have a type for it in C#, I don't see why we shouldn't use it